src/types/ApplicationSchema.ts

```diff
@@ -0,0 +1,54 @@
+import { ajv } from '../ajv';
+import { getLogger } from '../logger';
+import { applicationSchemaFieldSchema } from './ApplicationSchemaField';
+import type { JSONSchemaType } from 'ajv';
+import type { ApplicationSchemaField } from './ApplicationSchemaField';
+
+const logger = getLogger(__filename);
+
+export interface ApplicationSchema {
+  id: number;
+  // As of this commit, only one Application Schema version per Opportunity is
+  // visible through the API even though multiple are supported by the DB.
+  // Therefore it is be superfluous to expose it here or now. The latest schema
+  // will be the one retrieved for a given opportunity while the schema used by
+  // a particular application will be related to the schema used at the time.
+  // Nevertheless, we will declare the maximal fields here as a rule and where
+  // it is not needed a utility type can be defined instead, subtracting it.
+  version: number;
+  fields: ApplicationSchemaField[];
+  createdAt: Date;
+}
+
+export const applicationSchemaSchema: JSONSchemaType<ApplicationSchema> = {
```

Comment:
Oh no how awkward -- maybe we should call `applicationSchema` something different at the db level...  (We don't have to feel locked into the ERD; I think it makes sense to avoid developer terms of art in our table names)

Reply:
Application Form? Application Fields? Application List? Application Items?

Reply:
I think Application Form seems like a good idea